lte/gateway/c/core/oai/tasks/s11/s11_ie_formatter.cpp

```diff
@@ -948,15 +954,15 @@ status_code_e gtpv2c_failed_bearer_context_within_delete_bearer_request_ie_set(
 //------------------------------------------------------------------------------
 status_code_e gtpv2c_ebis_within_delete_bearer_request_ie_set(
     nw_gtpv2c_msg_handle_t* msg, const ebis_to_be_deleted_t* ebis_tbd) {
-  int rc = RETURNok;
+  nw_rc_t rc = NW_OK;
```

Comment:
Change the prototype

Reply:
This prototype is independent of this return type.